samples/core/Querying/SqlQueries/Program.cs

```diff
@@ -49,128 +50,104 @@ from [Post] as [p]
 
         using (var context = new BloggingContext())
         {
-            #region FromSql
```

Comment:
Remove this file if it's no longer being referenced.

Reply:
#4931